components/admin_console/admin_console.jsx

```diff
@@ -73,10 +73,20 @@ export default class AdminConsole extends React.Component {
     }
 
     renderRoutes = (extraProps) => {
-        const firstUrl = Object.values(AdminDefinition.reporting)[0].url;
+        const firstUrl = Object.values(AdminDefinition.about)[0].url;
         return (
             <Switch>
-                {Object.values({...AdminDefinition.reporting, ...AdminDefinition.other}).map((item) => {
+                {Object.values({
```

Comment:
You here are removing the whole nesting thing so you are redirecting any invalid url to the first "about" section entry. The previous implementation using the renderSectionRoutes was handling that. Another possible solution to this would be something like:
```js
{Object.values(AdminDefinition).map(this.renderSectionRoutes.bind(this, extraProps))}
```
That inside the switch + the Redirect at the end, should work properly and as expected (I haven't tried)

Reply:
I see that I still have a routing issue.